src/shared/components/MemoryInput.js

```diff
@@ -1,36 +0,0 @@
-import React from 'react';
```

Comment:
We do have 2 files with almost identical function Memory Input. Can we combine them?
src/shared/ResourceForm/fields/RuntimeResources.js
src/resources/Namespaces/MemoryQuotas.js